### DISCUSSION TITLE
Auto import

### DISCUSSION THREAD

**Discussion by monkz88** [rocksdanister/lively#506] [Q&A]
Hi,

Let's say we have a folder that's synced with onedrive. Is it possible that whenever this folder gets a new image or video, that this gets imported to lively and the ones not present anymore in the folder, gets removed? So lively only play whatever is in this folder?

Best regards

**Comment by rocksdanister** [comment]
It should be possible by writing a autohotkey  script using lively commandline controls:
https://github.com/rocksdanister/lively/wiki/Command-Line-Controls
